tests/test_recovery_tasks.py

```
@@ -0,0 +1,378 @@
+        # Recent documents should not be recovered
+        assert result["found"] == 0
+
+
+# =============================================================================
+# WAL REPLAY TESTS
+# =============================================================================
+
+class TestWALReplayTask:
+    """Tests for WAL replay task"""
+
+    def test_replay_pending_wal_entries(self, mock_supabase):
```

Ruff (0.14.13)

Unused method argument: `mock_supabase`

(ARG002)